app/src/test/java/com/github/multimatum_team/multimatum/DeadlineDetailsTest.kt

```diff
@@ -57,9 +61,8 @@ class DeadlineDetailsTest {
 
     @Test
     fun `Given a deadline not yet due or done, the activity should display it`() {
-        val intent = DeadlineDetailsActivity.newIntent(ApplicationProvider.getApplicationContext(),
-            "1", Deadline("Test 1", DeadlineState.TODO, clockService.now().plusDays(7))
-        )
+        val intent = DeadlineDetailsActivity.newIntent(ApplicationProvider.getApplicationContext(), "0" )
```

Comment:
What is this `"0"` argument?

Reply:
Now, the intent send only the id of the deadline, which is the position in the list of MockDeadlineRepository at the end of the test class. As we want to check the firt deadline of the list, we put 0 (like every list in coding, we begin a list at 0)